cordova-plugin-mobilespec-tests/tests/datauri.tests.js

```diff
@@ -27,6 +27,7 @@ exports.defineAutoTests = function () {
 
         function onMessage(done, msg) {
             if (msg.data == 'foo') {
+                expect('Suppress SPEC HAS NO EXPECTATIONS').toBeDefined();
```

Comment:
without it, the tests show the "Spec has no Expectation" warnings. this fixes it.
see http://stackoverflow.com/questions/27016235/angular-js-unit-testing-httpbackend-spec-has-no-expectations